content/guides/azure-todo-static-web-app.md

```diff
@@ -0,0 +1,747 @@
+
+What makes Azure Static Web Apps particularly compelling for developers is its built-in CI/CD pipeline powered by [Github Actions](https://github.com/features/actions). When you connect your repository, Azure automatically configures the necessary Github workflows – push your code and watch as Github Actions builds, optimizes, and deploys your entire application across a global network
+
+In this guide, we'll show you how to build a simple todo application using Azure Static Web Apps. You'll learn the basics of getting your website online and creating your first API endpoint with Azure Functions. By the end of this tutorial, you'll understand how to combine static web content with dynamic features to create a fully functional web application.
+
+## Prerequisites
+
+Before we begin, make sure you have:
+
+- [Node.js](https://nodejs.org/) 18.x or later installed
+- [Visual Studio Code](https://code.visualstudio.com/) with the [Azure Static Web Apps](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps) and [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) extensions installed
+- An [Azure account](https://azure.microsoft.com/free/) with an active subscription
+- A [Neon account](https://console.neon.tech/signup) and project
+- [Azure Functions Core Tools version 4.x](https://learn.microsoft.com/en-gb/azure/azure-functions/create-first-function-vs-code-node?pivots=nodejs-model-v4#install-or-update-core-tools)
+- [Azure Static Web Apps CLI](https://www.npmjs.com/package/@azure/static-web-apps-cli)
+
+## Create a Neon Project
+
+Neon is now available in Azure! You can create serverless Postgres databases that run on Azure infrastructure. To learn more about Neon's Azure launch, check out the [announcement post](/blog/neon-is-coming-to-azure).
+
+To create Neon project on Azure, follow our [Getting Started with Neon on Azure guide](/guides/neon-azure-integration).
+
+Once created, save your database connection string, which you'll need to connect to your Neon Postgres database from Azure Functions.
+
+## Database Schema
+
+For our todo application, we'll need a simple database schema to store todo items. We'll create a `todos` table with the following fields:
+
+- `id`: Auto-incrementing unique identifier (Primary Key)
+- `text`: Required text field to store the task description
+- `completed`: Boolean field with a default value of false to track task completion status
+
+\*We will be creating the table via Azure Functions later in the guide.
```

Comment:
what's the \* for?

Reply:
is it a reference?

Reply:
I've added this as a small note to avoid using a larger admonition block. However, I'm happy to switch to an admonition if that would be preferable.